docs/run_tests.rst

```diff
@@ -0,0 +1,145 @@
+.. _topics-run_tests:
+
+=========
+Run Tests
+=========
+
+Running Tests
+=============
+
+ducktape discovers and runs tests in the path provided, here are some ways to run tests::
+
+    ducktape <relative_path_to_testdirectory>               # e.g. ducktape dir/tests
+    ducktape <relative_path_to_file>                        # e.g. ducktape dir/tests/my_test.py
+    ducktape <path_to_test>[::SomeTestClass]                # e.g. ducktape dir/tests/my_test.py::TestA
+    ducktape <path_to_test>[::SomeTestClass[.test_method]]  # e.g. ducktape dir/tests/my_test.py::TestA.test_a
+
+Options
+=======
+
+To see a complete listing of options run::
+
+    ducktape --help
+
+
+Here is a list of run options:
```

Comment:
Is this manually generated or auto-generated? Just curious what our maintenance cost will be as we add more config options.